code/modules/clothing/chameleon.dm

```diff
@@ -467,6 +477,8 @@
 	permeability_coefficient = 0.01
 	flags_cover = MASKCOVERSEYES | MASKCOVERSMOUTH
 	w_class = WEIGHT_CLASS_SMALL
+	special_desc_requirement = EXAMINE_CHECK_SYNDICATE // Skyrat edit
+	special_desc = "A chameleon mask employed by the Syndicate in infiltration operations." // Skyrat edit
```

Comment:
"A chameleon mask employed by the Syndicate in infiltration operations. There is a built in voice changer that connects to the user's worn ID."